### ISSUE TITLE
Volume for plugins

### ISSUE THREAD

**Issue by almereyda** [archivy/archivy-docker#19]
There should be a way to persist plugins installed into the container environment. The README suggests that this is currently not supported.

Some suggestions can be taken from:

- https://testdriven.io/blog/docker-best-practices/

Ideally plugins are not installed into the container, but preinstalled into custom images, that reuse the `archivy` images with a `FROM:` directive. This could be exemplified in the documentation alongside this repository.

**Comment by Uzay-G** [comment]
This seems like a good idea. I don't have much time to do this right now though, but I'll keep it in mind.

**Comment by almereyda** [comment]
Yes, I only wanted to leave a note, because the idea came to mind.
